examples/light-switch-app/efr32/src/binding-handler.cpp

```diff
@@ -38,17 +38,18 @@ void BoundDeviceChangedHandler(const EmberBindingTableEntry * binding, chip::Dev
         ChipLogError(NotSpecified, "OnOff command failed: %" CHIP_ERROR_FORMAT, error.Format());
     };
 
-    if (binding->type == EMBER_MULTICAST_BINDING)
+    if (binding.type == EMBER_MULTICAST_BINDING)
     {
         ChipLogError(NotSpecified, "Group binding is not supported now");
         return;
     }
 
-    if (binding->type == EMBER_UNICAST_BINDING && binding->local == 1 && binding->clusterId == Clusters::OnOff::Id)
+    if (binding.type == EMBER_UNICAST_BINDING && binding.local == 1 && binding.clusterId.HasValue() &&
+        binding.clusterId.Value() == Clusters::OnOff::Id)
```

Comment:
@mkardous-silabs This is wrong.  If there is no cluster id, it means "bind all clusters", which is not what this is implementing....

Reply:
fixed in #15515